concent_api/signing_service_integration_test_middleman.py

```diff
@@ -75,6 +91,80 @@ def test_case_send_golem_message_frame_with_correct_transaction_signing_request(
     assert_condition(type(response.payload), SignedTransaction, f'Deserialized response payload type is {type(response.payload)} instead of SignedTransaction.')
 
 
+@count_fails
+def test_case_2_send_golem_message_frame_with_incorrect_transaction_signing_request_do_not_kill_communication(config: ConfigParser, **kwargs: Any) -> None:
+    """
+    1. Client sends GolemMessageFrame with incorrect TransactionSigningRequest to MiddleMan.
+    2  Client sends GolemMessageFrame with correct TransactionSigningRequest to MiddleMan.
+    3. MiddleMan responds with GolemMessageFrame with SignedTransaction.
+    """
+
+    # Create GolemMessageFrame with incorrect TransactionSigningRequest.
+    golem_message_frame = create_golem_message_frame_with_incorrect_transaction_signing_request(
+        request_id=get_current_utc_timestamp(),
+    )
+
+    # Send message through wrapper and do not wait for response.
+    send_message_to_middleman_without_response(
+        message=golem_message_frame,
+        config=config,
+        concent_private_key=CONCENT_PRIVATE_KEY,
+    )
+
+    # Create GolemMessageFrame with correct TransactionSigningRequest.
+    golem_message_frame = create_golem_message_frame_with_correct_transaction_signing_request(
+        request_id=get_current_utc_timestamp(),
+    )
+
+    # Send message through wrapper and receive deserialized response.
+    response = send_message_to_middleman_and_receive_response(
+        message=golem_message_frame,
+        config=config,
+        concent_private_key=CONCENT_PRIVATE_KEY,
+        concent_public_key=CONCENT_PUBLIC_KEY,
+    )
+
+    # Check response.
+    assert_condition(type(response), GolemMessageFrame, f'Deserialized response type is {type(response)} instead of GolemMessageFrame.')
+    assert_condition(type(response.payload), SignedTransaction, f'Deserialized response payload type is {type(response.payload)} instead of SignedTransaction.')
```

Comment:
Please also here break lines of these conditions

Reply:
:+1: done